benchmarks/graphics_pipeline/main.cpp

```diff
@@ -219,14 +227,31 @@ void ProjApp::InitKnobs()
     PPX_ASSERT_MSG(!cl_options.HasExtraOption("vs-shader-index"), "--vs-shader-index flag has been replaced, instead use --vs and specify the name of the vertex shader");
     PPX_ASSERT_MSG(!cl_options.HasExtraOption("ps-shader-index"), "--ps-shader-index flag has been replaced, instead use --ps and specify the name of the pixel shader");
 
-    pKnobVs = GetKnobManager().CreateKnob<ppx::KnobDropdown<std::string>>("vs", 0, kAvailableVsShaders);
-    pKnobVs->SetDisplayName("Vertex Shader");
+    mCurrentSceneIndex = 0;
+    pCurrentScene = GetKnobManager().CreateKnob<ppx::KnobDropdown<std::string>>("scene", mCurrentSceneIndex, kAvailableScenes);
+    pCurrentScene->SetDisplayName("Scene");
 
-    pKnobPs = GetKnobManager().CreateKnob<ppx::KnobDropdown<std::string>>("ps", 0, kAvailablePsShaders);
-    pKnobPs->SetDisplayName("Pixel Shader");
+    pMaterialsKnobVs.resize(kMaxMaterialCount );
```

Comment:
any reason for the extra space after kMaxMaterialCount here, on the next 2 lines?